src/check-signature.ts

```diff
@@ -6,17 +6,21 @@ import debug from 'debug';
 const d = debug('electron-notarize');
 
 const codesignDisplay = async (opts: NotaryToolNotarizeAppOptions) => {
-  const result = await spawn('codesign', ['-dv', '-vvvv', '--deep', path.basename(opts.appPath)], {
-    cwd: path.dirname(opts.appPath),
-  });
+  const result = await spawn(
+    'codesign',
+    ['-dv', '-vvvv', '--deep', `./${path.basename(opts.appPath)}`],
```

Comment:
Just curious, why basename relative to root? Why not `path.resolve(opts.appPath)`?